test/tests/postprocessors/pseudotimestep/tests

```diff
@@ -0,0 +1,10 @@
+[Tests]
+  design = 'PseudoTimestep.md'
+  issues = '#25931'
+  [pseudotimestep]
+    type = 'CSVDiff'
+    input = 'steady_state.i'
```

Comment:
Can you use cli_args to test all of the different pseudotimesteppers?  The other methods need tested.

Reply:
`cli-args` is a good idea, but it doesn't fit with the argument list. I was about to add independent tests for every method; this was a draft submission to check the status with CIVET before I extended it.